src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java

```diff
@@ -92,4 +89,10 @@ public double getTurnSpeed() {
 			* NathanGain.TURN_SPEED_SCALE;
 		return turnSpeed;
 	}
+
+	private void bindDpadCommand(Button direction, double angle, Command normalDrive) {
+		direction.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, angle,
+			RobotMap.Component.navx, RobotMap.Component.chassisDriveMC));
```

Comment:
This was an existing error, but `RobotMap.Component.chassisDriveMC` refers to distance PID using encoders. This should be `RobotMap.Component.chassisTurnMC`, which utilizes the NavX as a sensor.

I don't believe it will work at all as is, unless at some point `chassisDriveMC` was incorrectly set up to use the NavX.